webpack.config.js

```diff
@@ -18,9 +18,113 @@ const PATHS = {
   ROOT: Path.resolve(),
   SRC: Path.resolve('client/src'),
   DIST: Path.resolve('client/dist'),
+  DIST_JS: Path.resolve('client/dist/js'),
   THIRDPARTY: Path.resolve('thirdparty'),
 };
 
+// client/dist/js/jquery-validation/localizaton
+const locales = {
```

Comment:
Is there a reason we can't just copy the whole directories?

Reply:
yeah there's all the unminified versions as well, wasteful and makes for an awful diff